### ISSUE TITLE
"Create physical bones" yields incorrect collision shapes for bones

### ISSUE THREAD

**Issue by and3rson** [godotengine/godot#24836]
**Godot version:**
3.1-alpha5

**OS/device including version:**
Arch Linux rolling + Intel & NVIDIA

**Issue description:**
"Create physical bones" creates all bones with wrong rotation & translation. Scale seems to be correct.

**Steps to reproduce:**
1. Import the provided model
2. Use "Create physical bones" command for skeleton.

**Actual result:**
![Screenshot 1](https://dun.ai/p/gxD3Jq9.jpg)
![Screenshot 2](https://dun.ai/p/8ozJ2Rr.jpg)

**Minimal reproduction project:**
[issue-demo.zip](https://github.com/godotengine/godot/files/2736396/issue-demo.zip)

**Comment by and3rson** [comment]
@AndreaCatania just a quick heads up - let me know if you need any help with reproducing/debugging this thing.

I've tried writing my own small editor plugin that would create physical bones & collision and looks like in my case creating CapsuleShape as child for each PhysicalBone and simply rotating it by 90 degrees yielded correct alignment of collision shape to the bone.

**Comment by and3rson** [comment]
:'(

**Comment by akien-mga** [comment]
Still reproducible in the current master branch.

**Comment by jedStevens** [comment]
I've fixed this by adjusting the rotation and position of the PhysicalBone's body; rotation by 90 degrees over local x and changing the position like so `(x,y,z) = (x,-z,y)`. I could submit a pr but my fork of godot is not well maintained.

Here is the file I changed: `editor/plugins/skeleton_editor.cpp:116`
OLD
```
116	Transform body_transform;
117	body_transform.origin = Vector3(0, 0, -half_height);
```

NEW
```
116	Transform body_transform;
117	body_transform.origin = Vector3(0, half_height, 0);
118	body_transform.basis = body_transform.rotated(body_transform.basis[0], 3.141593 / 2.0).basis;
```

Not sure how to cleanly share this, or if my code can be done in a better way, but this seems to fix this issue.

**Comment by bojidar-bg** [comment]
> I could submit a pr

By all means do so :smiley:

**Comment by jedStevens** [comment]
Okk I'll have it in a sec

**Comment by AndreaCatania** [comment]
Fixed by #36008

![Screenshot from 2020-02-09 09-49-55](https://user-images.githubusercontent.com/8342599/74099228-94c27f80-4b21-11ea-940d-84d97ecbf582.png)
